cpp/add_isbns_or_issns_to_articles.cc

```diff
@@ -96,15 +97,15 @@ void PopulateParentIdToISBNAndISSNMap(
         std::vector<size_t> _029_field_indices;
         record.getFieldIndices("029", &_029_field_indices);
         for (const auto _029_field_index : _029_field_indices) {
-            const Subfields subfields(fields[_029_field_index]);
+            const Subfields &subfields(record.getSubfields(_029_field_index));
```

Comment:
We always want to use a reference here, I thing: http://stackoverflow.com/questions/7340925/return-by-value-assigned-to-const-reference